### PR TITLE
Add custom group claim

### PR DESCRIPTION
**Pull request by Klucherev**
Some OIDC providers don't have "groups" claim in id token, but have "roles". With this setting in traccar.xml there is ability to change it to work properly.

Example is
` <entry key='openid.groupsClaimName'>roles</entry>`

I didn't find how to add new commits to PR so opened new one

### CODE REVIEW

**Comment by Klucherev** [comment]
@tananaev Please check

File: src/main/java/org/traccar/database/OpenIdProvider.java [hunk 1]
```diff
@@ -89,11 +91,12 @@ public OpenIdProvider(Config config, LoginService loginService, HttpClient httpC
...
 
+
         if (config.hasKey(Keys.OPENID_ISSUER_URL)) {
             HttpRequest httpRequest = HttpRequest.newBuilder(
-                URI.create(config.getString(Keys.OPENID_ISSUER_URL) + "/.well-known/openid-configuration"))
-                .header("Accept", "application/json")
-                .build();
+                            URI.create(config.getString(Keys.OPENID_ISSUER_URL) + "/.well-known/openid-configuration"))
+                    .header("Accept", "application/json")
+                    .build();
```
Comment (tananaev) [inline src/main/java/org/traccar/database/OpenIdProvider.java:99]: Please undo all the formatting changes.

File: src/main/java/org/traccar/database/OpenIdProvider.java [hunk 2]
```diff
@@ -89,11 +91,12 @@ public OpenIdProvider(Config config, LoginService loginService, HttpClient httpC
         baseUrl = new URI(WebHelper.retrieveWebUrl(config));
         callbackUrl = new URI(WebHelper.retrieveWebUrl(config) + "/api/session/openid/callback");
 
+
```
Comment (tananaev) [inline src/main/java/org/traccar/database/OpenIdProvider.java:94]: And this also

**Review by Klucherev — COMMENTED** [review]
Removed extra spaces

**Comment by tananaev** [comment]
It doesn't look like you fixed all the places.

**Comment by Klucherev** [comment]
> It doesn't look like you fixed all the places.

Sorry, i think is is done now

**Comment by tananaev** [comment]
Merged, thank you.

**Pull request merged**